4.4/debian-10/rootfs/opt/bitnami/scripts/libmongodb.sh

```diff
@@ -802,6 +829,32 @@ mongodb_configure_secondary() {
     fi
 }
 
+########################
+# Configure hidden node
+# Globals:
+#   None
+# Arguments:
+#   $1 - node
+# Returns:
+#   None
+#########################
+mongodb_configure_hidden() {
+    local -r node="${1:?node is required}"
+
+    mongodb_wait_for_primary_node "$MONGODB_INITIAL_PRIMARY_HOST" "$MONGODB_INITIAL_PRIMARY_PORT_NUMBER" "$MONGODB_INITIAL_PRIMARY_ROOT_USER" "$MONGODB_INITIAL_PRIMARY_ROOT_PASSWORD"
+
+    if mongodb_node_currently_in_cluster "$node"; then
+        info "Node currently in the cluster"
+    else
+        info "Adding node to the cluster"
```

Comment:
```suggestion
        info "Adding hidden node to the cluster"
```
At this point, I would add something to differentiate if it is a hidden node or not